numpy/ma/tests/test_core.py

```diff
@@ -4376,16 +4376,27 @@ def test_getitem(self):
                                    [1, 0, 0, 0, 0, 0, 0, 0, 1, 0])),
                           dtype=[('a', bool), ('b', bool)])
         # No mask
-        self.assertTrue(isinstance(a[1], MaskedArray))
+        assert_equal(type(a[1]), mvoid)
```

Comment:
Perhaps also check equality of `_data` and `_mask` as done for element `0` below?